libs/detectors/edgetpu/mobilenet_ssd.py

```diff
@@ -50,6 +50,7 @@ def __init__(self, config):
         # Get class id from config
         self.class_id = int(self.config.get_section_dict('Detector')['ClassID'])
         self.score_threshold = float(self.config.get_section_dict('Detector')['MinScore'])
+        self.face_classifier = 'Classifier' in self.config.get_sections()
```

Comment:
Maybe a name like `has_face_classifier` makes it clearer that this is a boolean?